<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AnaliticWindow</class>
 <widget class="QMainWindow" name="AnaliticWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>701</width>
    <height>485</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Аналитическая модель</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>672</width>
      <height>689</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="font">
            <font>
             <family>Nimbus Roman No9 L</family>
             <pointsize>14</pointsize>
             <weight>75</weight>
             <italic>true</italic>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>    Березин Иван Сергеевич ИУ5-13М</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QGridLayout" name="gridLayout">
           <item row="3" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>N</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>To</string>
             </property>
            </widget>
           </item>
           <item row="7" column="0">
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>Tk2</string>
             </property>
            </widget>
           </item>
           <item row="9" column="0">
            <widget class="QLabel" name="label_12">
             <property name="text">
              <string>Тпр</string>
             </property>
            </widget>
           </item>
           <item row="8" column="0">
            <widget class="QLabel" name="label_11">
             <property name="text">
              <string>C</string>
             </property>
            </widget>
           </item>
           <item row="11" column="0">
            <widget class="QLabel" name="label_14">
             <property name="text">
              <string>Tdi</string>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Tp</string>
             </property>
            </widget>
           </item>
           <item row="10" column="0">
            <widget class="QLabel" name="label_13">
             <property name="text">
              <string>m</string>
             </property>
            </widget>
           </item>
           <item row="6" column="0">
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>Tk1</string>
             </property>
            </widget>
           </item>
           <item row="5" column="1" colspan="2">
            <widget class="QLineEdit" name="tGenerationRequest"/>
           </item>
           <item row="12" column="0">
            <widget class="QLabel" name="label_15">
             <property name="text">
              <string>P</string>
             </property>
            </widget>
           </item>
           <item row="10" column="1" colspan="2">
            <widget class="QSpinBox" name="quantityDisk">
             <property name="value">
              <number>1</number>
             </property>
            </widget>
           </item>
           <item row="9" column="1" colspan="2">
            <widget class="QLineEdit" name="tProcessingOnCPU"/>
           </item>
           <item row="7" column="1" colspan="2">
            <widget class="QLineEdit" name="tTransmitOpposite"/>
           </item>
           <item row="4" column="1" colspan="2">
            <widget class="QLineEdit" name="tDoProcessing"/>
           </item>
           <item row="12" column="1" colspan="2">
            <widget class="QLineEdit" name="pAfterProcessingOnDiskToCPU"/>
           </item>
           <item row="11" column="1" colspan="2">
            <widget class="QLineEdit" name="tProcessingOnDisk"/>
           </item>
           <item row="6" column="1" colspan="2">
            <widget class="QLineEdit" name="tTransmitForward"/>
           </item>
           <item row="3" column="1" colspan="2">
            <widget class="QSpinBox" name="quantityPC">
             <property name="maximum">
              <number>500</number>
             </property>
             <property name="value">
              <number>10</number>
             </property>
            </widget>
           </item>
           <item row="8" column="1" colspan="2">
            <widget class="QSpinBox" name="quantityCPU">
             <property name="value">
              <number>1</number>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label">
             <property name="font">
              <font>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Исходные данные</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1" colspan="2">
            <widget class="QPushButton" name="pushButton">
             <property name="text">
              <string>Расшифровка</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="1" column="0">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>K1</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>K2                        </string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="K1"/>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>Точность                      </string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLineEdit" name="delta"/>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="K2"/>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_29">
             <property name="font">
              <font>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>   Коэффициенты</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="modeling">
           <property name="text">
            <string>Начать моделирование</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="13" column="0" colspan="3">
        <widget class="QTextEdit" name="listDisks">
         <property name="maximumSize">
          <size>
           <width>350</width>
           <height>82</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="4" column="1" colspan="2">
        <widget class="QLineEdit" name="loadingCPU"/>
       </item>
       <item row="10" column="0">
        <widget class="QLabel" name="label_26">
         <property name="text">
          <string>Количество итераций</string>
         </property>
        </widget>
       </item>
       <item row="12" column="1">
        <widget class="QSpinBox" name="numberDisk">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="value">
          <number>1</number>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_17">
         <property name="text">
          <string>pPC</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_18">
         <property name="text">
          <string>pпол</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1" colspan="2">
        <widget class="QLineEdit" name="loadingClient"/>
       </item>
       <item row="10" column="1" colspan="2">
        <widget class="QLineEdit" name="iteration"/>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_22">
         <property name="text">
          <string>Tr</string>
         </property>
        </widget>
       </item>
       <item row="9" column="1" colspan="2">
        <widget class="QLineEdit" name="lf"/>
       </item>
       <item row="1" column="1" colspan="2">
        <widget class="QLineEdit" name="loadingPC"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_19">
         <property name="text">
          <string>pk</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1" colspan="2">
        <widget class="QLineEdit" name="reactSys"/>
       </item>
       <item row="3" column="1" colspan="2">
        <widget class="QLineEdit" name="loadingChanel"/>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="label_23">
         <property name="text">
          <string>lymda_0</string>
         </property>
        </widget>
       </item>
       <item row="12" column="2">
        <widget class="QPushButton" name="request">
         <property name="text">
          <string>Запросить</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1" colspan="2">
        <widget class="QLineEdit" name="tCycle"/>
       </item>
       <item row="9" column="0">
        <widget class="QLabel" name="label_24">
         <property name="text">
          <string>lymda_k</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1" colspan="2">
        <widget class="QLineEdit" name="lf0"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_20">
         <property name="text">
          <string>pпр</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="3">
        <widget class="QLabel" name="label_16">
         <property name="maximumSize">
          <size>
           <width>350</width>
           <height>15</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>  Результаты моделирования</string>
         </property>
        </widget>
       </item>
       <item row="12" column="0">
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <widget class="QLabel" name="label_28">
           <property name="text">
            <string>Номер диска</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_21">
         <property name="text">
          <string>Tц</string>
         </property>
        </widget>
       </item>
       <item row="14" column="0" colspan="3">
        <widget class="QPushButton" name="cleaning">
         <property name="text">
          <string>Очистить</string>
         </property>
        </widget>
       </item>
       <item row="11" column="0" colspan="3">
        <widget class="QLabel" name="label_27">
         <property name="maximumSize">
          <size>
           <width>350</width>
           <height>17</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>             Загрузка дисков</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_25">
         <property name="text">
          <string>n</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1" colspan="2">
        <widget class="QLineEdit" name="U"/>
       </item>
       <item row="15" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>701</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
